app/models/jwt_identifier.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Laboratory: represents a lab result
+class JwtIdentifier < ApplicationRecord
+  belongs_to :application, class_name: 'OauthApplication'
+
+  # Finds all of the JWT identifiers that belong to JWTs that have since expired
+  # and therefore no longer need to be checked.
+  scope :purge_eligible, lambda {
+    where('expiration_date <= ?', Time.now)
```

Comment:
So I chose `Time` because it has good conversion from an Epoch timestamp with `Time.at` whereas DateTime.requires conversion to string first from Epoch timestamp it looks like, and all the expirations provided on JWTs will be epoch timestamps.

Reply:
That's a much stronger argument than my consistency ask, and makes sense to me.